### PR TITLE
feat: add search panel

### PR DESCRIPTION
**Pull request by evan-spidy**

### CODE REVIEW

File: src/components/ForYou/ActionButton/__test__/index.test.tsx [hunk 1]
```diff
@@ -1,40 +1,89 @@
...
+    expect(screen.getByText("5")).toBeInTheDocument(); // Comments
+    expect(screen.getByText("3")).toBeInTheDocument(); // Opens
+    expect(screen.getByTestId("confetti-mock")).toBeInTheDocument(); // Confetti mock presence
   });
 
-  it("triggers haptic feedback on like button click", () => {
-    render(<ActionButton />);
-    const likeButton = screen.getAllByRole("button")[0]; // Assuming the first button is the like button
-    fireEvent.click(likeButton);
+  // it("increments like count and triggers confetti and notification on like button click", () => {
```
Comment (Leon-pix-coder) [inline src/components/ForYou/ActionButton/__test__/index.test.tsx:64]: Is it necessary to keep it?

File: src/components/SceneLoading/__test__/index.test.tsx [hunk 2]
```diff
@@ -1,72 +1,90 @@
...
+
+  //   act(() => {
+  //     for (let i = 0; i < 30; i++) {
+  //       vi.advanceTimersByTime(1000); // Simulate sufficient time passing
+  //     }
+  //   });
+
+  //   expect(screen.getByTestId("cta-button")).toBeInTheDocument();
+  //   expect(mockSetIsLoading).toHaveBeenCalledWith(true);
+  // });
```
Comment (Leon-pix-coder) [inline src/components/SceneLoading/__test__/index.test.tsx:89]: Is it necessary to keep it?

**Review by Leon-pix-coder — APPROVED** [review]
all fine

File: src/components/ForYou/ActionButton/__test__/index.test.tsx (hunk 1 above)
Comment (evan-spidy) [reply src/components/ForYou/ActionButton/__test__/index.test.tsx:64]: will need it for later development

File: src/components/SceneLoading/__test__/index.test.tsx (hunk 2 above)
Comment (evan-spidy) [reply src/components/SceneLoading/__test__/index.test.tsx:89]: will need it for later development

**Pull request merged**